source/component/PasDoc_GenHtml.pas

```diff
@@ -2496,18 +2496,19 @@ function TGenericHTMLDocGenerator.FormatTableOfContents(
 { THTMLDocGenerator ---------------------------------------------------------- }
 
 function THTMLDocGenerator.MakeBodyBegin: string;
-
+Var
+  Skip : Boolean;
+  
   function MakeNavigation: string;
 
     function LocalMakeLink(const Filename, Caption: string): string;
     begin
-      Result := '<a href="' + EscapeURL(Filename) + '" class="navigation">' +
-        ConvertString(Caption) + '</a>';
+      Result := '<a href="' + EscapeURL(Filename) + '">' + ConvertString(Caption) + '</a>';
     end;
 
     function LocalMakeParagraphLink(const Filename, Caption: string): string; overload;
```

Comment:
Should probably be named `LocalMakeListItemLink` now

Reply:
I second that :)

Reply:
I'm agree with that

Reply:
You talk about the  function LocalMakeParagraphLink ??